src/ert/gui/simulation/simulation_panel.py

```diff
@@ -123,31 +123,53 @@ def runSimulation(self):
             f"Are you sure you want to use case '{case_name}' for initialization of "
             "the initial ensemble when running the simulations?"
         )
-        start_simulations = QMessageBox.question(
-            self, "Start simulations?", message, QMessageBox.Yes | QMessageBox.No
-        )
-
-        if start_simulations == QMessageBox.Yes:
-
-            arguments = self.getSimulationArguments()
-            dialog = RunDialog(
-                self._config_file,
-                create_model(
-                    self.ert,
-                    self.facade.get_ensemble_size(),
-                    self.facade.get_current_case_name(),
-                    arguments,
-                    str(uuid.uuid4()),
-                ),
+        if (
+            QMessageBox.question(
+                self, "Start simulations?", message, QMessageBox.Yes | QMessageBox.No
             )
-            self.run_button.setDisabled(True)
-            self.run_button.setText("Simulation running...")
-            dialog.startSimulation()
-            dialog.exec_()
-            self.run_button.setText("Start simulation")
-            self.run_button.setDisabled(False)
-
-            self.notifier.emitErtChange()  # simulations may have added new cases
+            == QMessageBox.Yes
+        ):
+            model = create_model(
+                self.ert,
+                self.facade.get_ensemble_size(),
+                self.facade.get_current_case_name(),
+                self.getSimulationArguments(),
+                str(uuid.uuid4()),
+            )
+
+            abort = False
+            if model.check_if_runpath_exists():
+                if (
+                    QMessageBox.warning(
+                        self,
+                        "Start simulations?",
+                        (
+                            "ERT is running in an existing runpath.\n\n"
```

Comment:
Could we also mention how many of the realizations are running in existing run paths? I think a common way to run us to use `Single realization test-run` followed by a full ensemble. So in that case say: `1 out of N realizations are running in existing run paths`

Reply:
So in this case N is ensemble size not active mask size?

Reply:
Will not implement this change in this PR as it requires quite some work to get the correct number of realizations. Created a new issue just for this. #4410